packages/cloud_functions/cloud_functions_platform_interface/lib/src/method_channel_cloud_functions.dart

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of cloud_functions_platform_interface;
+
+class CloudFunctionsException implements Exception {
+  CloudFunctionsException._(this.code, this.message, this.details);
+
+  final String code;
+  final String message;
+  final dynamic details;
+}
+
+class MethodChannelCloudFunctions extends CloudFunctionsPlatform {
```

Comment:
add doc comment saying this is the platform implementation backed by a `MethodChannel`